ci/centos-7/Dockerfile

```diff
@@ -17,13 +23,13 @@ RUN yum -y install \
   && yum clean all && rm -rf /var/cache/yum
 
 RUN yum -y install \
-    devtoolset-7 \
+    devtoolset-8 \
```

Comment:
We need to make sure to update the OS/Compiler support matrix for this change. Similar questions with all of the compiler version changes later.

Reply:
Not yet. Depending on feedback to this PR more changes may need to go into Spicy or spicy-plugin. Before the next Zeek release we should have cut releases of Spicy and spicy-plugin though, so I am leaving this open for now.

Reply:
This PR still requires docs changes, see [below](https://github.com/zeek/zeek/pull/2006#discussion_r865184038)